config/suppressions.xml

```diff
@@ -108,4 +108,6 @@
   <suppress id="noSourceforgeIoLinks" files="[\\/]sun_style.xml" lines="31"/>
   <suppress id="noSourceforgeIoLinks" files="[\\/]index.xml.vm"/>
 
+  <suppress checks="ThrowsCount" files="[\\/]XmlMetaReader.java$"/>
```

Comment:
it is not required ... as you do catch.

Reply:
No, you told me to rethrow exceptions, logging is not being done anymore.

Reply:
please give a link to what violation it is required, or use Xpath to suppress only certain violation (not in whole file)

Reply:
I see now:
```
    public static ModuleDetails read(InputStream moduleMetadataStream, ModuleType moduleType)
             throws ParserConfigurationException, IOException, SAXException {
```

Reply:
comment over suppression should be:
`<!-- there are 3 thrown and all of them could be caused by xml parsing -->`

Reply:
Done